XouDevSpec/Classes/features/ProfileScreen/ViewModel/ProfileViewModelProtocol.swift

```diff
@@ -0,0 +1,8 @@
+//
+//  ProfileViewModelProtocol.swift
+//  XouDevSpec
+//
+//  Created by Xander Schoeman on 2020/04/27.
+//
+
+import Foundation
```

Comment:
is this an empty class, if so , u should remove it

Reply:
Done and done.